src/openrct2/localisation/Localisation.cpp

```diff
@@ -35,9 +35,9 @@
 #include <iterator>
 #include <limits.h>
 
-char gCommonStringFormatBuffer[512];
-uint8_t gCommonFormatArgs[80];
-uint8_t gMapTooltipFormatArgs[40];
+thread_local char gCommonStringFormatBuffer[512];
```

Comment:
What requires this to be `thread_local`?

Reply:
It is written to before many of the draw string calls and probably other places such as text that appears within the viewport.

Reply:
Banners would be an example that uses this and its happening during rendering so it it needs to become stack local which would change a lot of code or just mark it thread_local like I did.